### ISSUE TITLE
Add Large Block Interface (LBI) support for BSAM read/write access method

### ISSUE THREAD

**Issue by dsh33782** [z390development/z390#316]
Previously RPI 2229 implemented LBI support for QSAM get/put access method.  See rt\bat\RUNRTQSAM.BAT test.
At the request of Melvyn I am opening this issue to document adding support for BSAM read/write LBI support where the variable length prefix for records changes from LL00 to LLLL with high bit on indicating LBI specified in DECB.

**Comment by dsh33782** [comment]
Today I finished creating new directory bsam with 4 new test programs to test bsam read and write i/o with and without large block support (LBI indicator set in dcbe).  This is in addition to qsam directory which tests LBI support for qsam get/put options.
I did not have to make any additional changes to sz390 java code as the same code added for qsam in RPI 2229 also supports bsam read/write.  I am sending zip of bsam directory and rt\bat\RUNRTBSAM.BAT to Melvyn for testing with his zvsam repro code.
